src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.Linux.cs

```diff
@@ -25,7 +25,7 @@ private ThreadPriorityLevel PriorityLevelCore
             }
             set
             {
-                throw new PlatformNotSupportedException();
+                throw new PlatformNotSupportedException(); // Why is this not supported? 
```

Comment:
From https://linux.die.net/man/2/setpriority:
>According to POSIX, the nice value is a per-process setting. However, under the current Linux/NPTL implementation of POSIX threads, the nice value is a per-thread attribute: different threads in the same process can have different nice values. Portable applications should avoid relying on the Linux behavior, which may be made standards conformant in the future. 
But since this is a Linux-specific implementation, it might be feasible.